tools/md5sum.ml

```diff
@@ -0,0 +1,4 @@
+let () = 
+  let file = Sys.argv.(1) in
```

Comment:
I think you should check if the user passed an argument, and print a usage message if he did not.
I understand it is "for internal use only" but still....